sheets/api.py

```diff
@@ -1399,7 +1400,9 @@ def update_coupon_delivery_statuses(self, assignment_status_map):
                     log.error(
```

Comment:
Can we change to `log.exception` and remove the `exception: %s` here? `log.exception` will also print the stack trace.